src/dra/75DA0.c

```diff
@@ -0,0 +1,61 @@
+#include "common.h"
+#include "dra.h"
+#include "objects.h"
+#include "sfx.h"
+
+void func_80115DA0(void) {
```

Comment:
Can you check if you can merge the content of this file upward too?

Reply:
unfortunately no, it didn't work

Reply:
func_80115DA0 has a jump table